store/src/java/com/zimbra/cs/account/ZAttrAccount.java

```diff
@@ -35006,7 +35078,7 @@ public void unsetPasswordBlockCommonEnabled() throws com.zimbra.common.service.S
      * @param attrs existing map to populate, or null to create a new map
      * @return populated map to pass into Provisioning.modifyAttrs
      *
-     * @since ZCS 8.9.0
+     * @since ZCS 9.1.0
```

Comment:
is this change desired?

Reply:
I didn't change this at all, but i don't know why this change appears. it was changed by @chinmay15 in https://github.com/Zimbra/zm-mailbox/commit/d27f5920a6920b84e8a81a81df1f45ea4658710a
But he seems forgot to run `ant generate-getters`